pkg/pillar/hypervisor/kvm.go

```diff
@@ -173,6 +173,16 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   driver = "virtconsole"
   chardev = "charserial1"
   name = "org.lfedge.eve.console.prime"
+
+{{- if .DomainConfig.EnableVncShimVM}}
```

Comment:
What about adding a test to check that this is actually activated like the other tests we already have for this template?

Reply:
Added it